camayoc/tests/qcs/test_scan.py

```diff
@@ -136,12 +198,25 @@ def test_restart(shared_client, cleanup, source, isolated_filesystem):
         5) Assert that the scan can be restarted
         6) Assert that he scan completes
     :expectedresults: A restarted scan completes
-    :caseautomation: notautomated
     """
-    pass
+    scan = prep_network_scan(source, cleanup, shared_client)
+    scan.create()
+    wait_until_state(scan, state='running')
+    scan.pause()
+    time.sleep(2)
+    assert scan.status() == 'paused'
+    scan.restart()
+    time.sleep(2)
+    wait_until_state(scan)
+    assert scan.status() == 'completed'
```

Comment:
If the wait_until_state start raising an issue for timeout then you can assume it is on the expected state so you can drop this and assume if no exception was raised then you are good to do